### PR TITLE
`treehouses shutdown` english (fixes #2048)

### LINKED ISSUE #2048: `treehouses shutdown` help cancel description
The `treehouses led shutdown` help description for cancel does not match with the shutdown cancel output.

`Press crtl+c to cancel` versus `ctrl+c to cancel`

----
Also spelling error.

`crtl` should be `ctrl`

### PR DESCRIPTION
**Pull request by rjpadilla**

### CODE REVIEW

File: modules/shutdown.sh [hunk 1]
```diff
@@ -3,15 +3,15 @@ function shutdown {
   checkroot
   case "$1" in
     "")
-      echo "Shutting down in 60 seconds. Press crtl+c to cancel"
+      echo "Shutting down in 60 seconds. Press ctrl+c to cancel"
```
Comment (JLKwong) [inline modules/shutdown.sh:6]: Add a period at the end to read:
>      echo "Shutting down in 60 seconds. Press ctrl+c to cancel."

File: modules/shutdown.sh [hunk 2]
```diff
@@ -3,15 +3,15 @@ function shutdown {
...
       sleep 60
       /sbin/shutdown now
        ;;
     "now")
       /sbin/shutdown now
       ;;
     "in")
-      echo "Shutting down in $2 seconds. Press crtl+c to cancel"
+      echo "Shutting down in $2 seconds. Press ctrl+c to cancel"
```
Comment (JLKwong) [inline modules/shutdown.sh:14]: Add a period at the end to read:
>      echo "Shutting down in $2 seconds. Press ctrl+c to cancel."

File: modules/shutdown.sh [hunk 3]
```diff
@@ -32,13 +32,13 @@ function shutdown_help {
   echo
   echo "Example:"
   echo "  $BASENAME shutdown"
-  echo "      System shutdown in 60 seconds. 'ctrl+c' to cancel"
+  echo "      System shutdown in 60 seconds. Press ctrl+c to cancel"
```
Comment (JLKwong) [inline modules/shutdown.sh:35]: Add a period at the end to read:
> "      System shutdown in 60 seconds. Press ctrl+c to cancel."

File: modules/shutdown.sh [hunk 4]
```diff
@@ -32,13 +32,13 @@ function shutdown_help {
...
   echo
   echo "  $BASENAME shutdown now"
   echo "      System shutdown immediately"
   echo
   echo "  $BASENAME shutdown in <time in seconds>"
-  echo "      System will shutdown after the specified time. 'ctrl+c' to cancel"
+  echo "      System will shutdown after the specified time. Press ctrl+c to cancel"
```
Comment (JLKwong) [inline modules/shutdown.sh:41]: Add a period at the end to read:
> "      System will shutdown after the specified time. Press ctrl+c to cancel."

**Review by JLKwong — CHANGES_REQUESTED** [review]

**Review by JLKwong — CHANGES_REQUESTED** [review]
On Line 22:
>        echo "Error: only now, in and force options are supported"

This should be revised to clarify that the options are options:
>        echo "Error: only the options 'now', 'in', and 'force' are supported"

**Review by JLKwong — APPROVED** [review]
Control is better than Corntol. 🌽

**Review by dogi — APPROVED** [review]

**Pull request merged**